.github/workflows/coverage.yml

```diff
@@ -0,0 +1,69 @@
+name: C++ and Python coverage
+
+on: 
+  schedule:
+    - cron: '0 0 * * 0'
```

Comment:
Can we stop it from running when nothing changed?

Reply:
With a messy mechanism, yes. But I don't think it is worth it, given that it runs so infrequently (1x per week)?

Reply:
Ok, I clearly don't understand the syntax. Thought this means once per day.

Reply:
The final 0 is day of the week.